python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_decorator.py

```diff
@@ -15,6 +15,7 @@ def fivetran_assets(
     name: Optional[str] = None,
     group_name: Optional[str] = None,
     dagster_fivetran_translator: Optional[DagsterFivetranTranslator] = None,
+    connector_selector_fn: Optional[ConnectorSelectorFn] = None,
```

Comment:
For the top user-facing API, IMO would be nice to have this rendered as `Callable[...]` explicitly, so a user doesn't have to click through to understand what a `ConnectorSelectorFn` is. For the rest of the use-cases, using the alias makes sense to keep code concise.

Reply:
Yeah I agree that we need to make it easy to understand what `ConnectorSelectorFn` is. For the context, `ConnectorSelectorFn` is documented and added to the API docs in the subsequent PR [here](https://github.com/dagster-io/dagster/pull/28021/files).

`dagster-airlift` uses a custom type for it dag selector - for consistency, I would keep this as-is for now, but I'm open to revert both selectors to `Callable[...]` in the future if this is confusing to users.

Reply:
Looking at docs [here](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_type_aliases) about type aliases in Sphinx, it looks like `ConnectorSelectoFn` will be rendered as `Callable[...]`.